src/core/client/ui/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,138 @@
+import cn from "classnames";
+import React, { CSSProperties, isValidElement } from "react";
+import { Manager, Popper, Reference, RefHandler } from "react-popper";
+import AriaInfo from "../AriaInfo";
+import * as styles from "./Popover.css";
+
+type Placement =
+  | "auto-start"
+  | "auto"
+  | "auto-end"
+  | "top-start"
+  | "top"
+  | "top-end"
+  | "right-start"
+  | "right"
+  | "right-end"
+  | "bottom-end"
+  | "bottom"
+  | "bottom-start"
+  | "left-end"
+  | "left"
+  | "left-start";
+
+interface InnerProps {
```

Comment:
Maybe call it PopoverProps, we used InnerProps because of HOCs, we don't have one here :-)